docs/overrides/home.html

```diff
@@ -0,0 +1,151 @@
+			<h1>Take handwritten notes with ease.</h1>
+			<h2>From solving a physics problem to making travel diaries.</h2>
+		</div>
+		<p>Xournal++ is an open-source and cross-platform note-taking software that is fast, flexible, and functional. A modern rewrite and a more feature-rich version of the wonderful <span><a class ="Xournal-link" href="https://sourceforge.net/projects/xournal/">Xournal</a></span> program.</p>
+		<img src="img/Physics Problem.png">
+		<div class="download-hyperlink upper-button">
+			<a href="{{ 'installation/' | url }}">Download Xournal++</a>
+		</div>
+	</div>
+	
+	<div class="feature-list">
+		<div class="feature feature-right">
+		    <div class="feature-text">
+		        <h2>Handwrite your notes</h2>
+		        <div class="feature-description-container">
+		        	<div class="feature-description">
+		        		<p class="bullet-point">|</p><p>Support for pressure-sensitive stylus and drawing tablets (Wacom, Huion, XP-Pen, etc.)</p>
+		        	</div>
+		        </div>
+		        <div class="feature-description-container">
+		        	<div class="feature-description last-description">
+						<p class="bullet-point">|</p><p>Robust and customizable pen, highlighters, and eraser tool, allowing you to write how you want to write</p>
+					</div>
+				</div>
+				<div class="feature-link">
+					<a href="#">Learn more</a>
+				</div>
+		    </div>
+		    <img src="/img/Handwritten.png">
+		</div>
+		
+		<div class="feature feature-left">
+		    <div class="feature-text">
+		        <h2>Keep your work organized</h2>
+		        <div class="feature-description-container">
+		        	<div class="feature-description">
+		        		<p class="bullet-point">|</p><p>Use layers to make complex notes that are still pleasant to work with</p>
+		        	</div>
+		        </div>
+		        <div class="feature-description-container">
+		        	<div class="feature-description last-description">
+						<p class="bullet-point">|</p><p>The page previews and PDF bookmarks allow for easy navigation of your notes</p>
+	        		</div>
+	        	</div>
+	        	<div class="feature-link">
+					<a href="#">Learn more</a>
+				</div>
+		    </div>
+		    <img src = "/img/Layers.gif">
+		</div>
+		
+		<div class="feature feature-right">
+		    <div class="feature-text">
+		        <h2>Enhance and accelerate your note-taking</h2>
+		        <div class="feature-description-container">
+		        	<div class="feature-description">
+						<p class="bullet-point">|</p><p>Add images and create various shapes, from circles to splines to axis</p>
+					</div>
+				</div>
+				<div class="feature-description-container">
+					<div class="feature-description last-description">
+	        			<p class="bullet-point">|</p><p>Snap objects to rectangular grid or degrees of rotation</p>
+	        		</div>
+	        	</div>
+	        	<div class="feature-link">
+					<a href="#">Learn more</a>
+				</div>
+		    </div>
+		    <img src="/img/Create Shape.png">
```

Comment:
You should use a grid background for any drawings that resemble mathematical content.